src/gui/mzroll/scatterplot.cpp

```diff
@@ -720,4 +728,8 @@ void ScatterPlot::showPeakTable() {
     _peakTable->showAllGroups();
     _peakTable->setVisible(btnPeakTable->isChecked());
 
+}
+
+void ScatterPlot::peakTableVisibilityChanged(bool visible) {
```

Comment:
the signal could directly connect to btnPeakTable's setChecked, no?

Reply:
Right. Should have checked if `setChecked` was a slot.